lib/session.ex

```diff
@@ -0,0 +1,198 @@
+defmodule Janus.Session do
+  @moduledoc """
+  Keeps information about session and sends keep-alive messages
+  if given connection's transport requires it.
+  """
+
+  use GenServer
+  alias Janus.Connection
+
+  @default_timeout 5000
+
+  @type t :: pid()
+  @type message_t :: map()
+  @type connection_t :: pid()
```

Comment:
Type `pid` does not include named processes - please use either `GenServer.server` or `Process.dest` (here most likely the first one)
Also, you can move this type to `Janus.Connection` and reference here as `Connection.t()`